src/editors/playgroundController.ts

```diff
@@ -537,6 +532,26 @@ export default class PlaygroundController {
     return true;
   }
 
+  async _ensureConnection(): Promise<boolean> {
+    if (this._connectionController.isCurrentlyConnected()) {
+      return true;
+    }
+    const action = await vscode.window.showInformationMessage(
```

Comment:
This looks good but just UX-wise, maybe we should just get them to do it right away?